src/mcts/params.cc

```diff
@@ -485,6 +495,10 @@ SearchParams::SearchParams(const OptionsDict& options)
           options.Get<int>(kMaxCollisionVisitsScalingStartId)),
       kMaxCollisionVisitsScalingEnd(
           options.Get<int>(kMaxCollisionVisitsScalingEndId)),
+      kMoveSelectionVisitsScalingPower(
```

Comment:
Moving these two below kMaxCollisionVisitsScalingPower should get rid of warning

Reply:
Thanks, that worked!